sklearn/linear_model/least_angle.py

```diff
@@ -587,14 +587,15 @@ class Lars(LinearModel, RegressorMixin):
     sklearn.decomposition.sparse_encode
 
     """
+    method = 'lar'
```

Comment:
`self.method` is it used in another place than l678 and l699. 
I don't like defined like that.